<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="task1">
        <property name="text">
         <string>task 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="task2">
        <property name="text">
         <string>task 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="task3">
        <property name="text">
         <string>task 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="task4">
        <property name="text">
         <string>task 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="task5">
        <property name="text">
         <string>task 5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
